math/exprtk/Portfile

```diff
@@ -0,0 +1,26 @@
+PortSystem                  1.0
+PortGroup                   github 1.0
+
+github.setup                ArashPartow exprtk 0.0.3
+github.tarball_from         archive
+categories                  math devel
+maintainers                 openmaintainer
```

Comment:
what I asked for is to add "openmaintainer" in addition to your maintainer information. On its own this is invalid and it just means there is no maintainer and then it should be "nomaintainer"